app/app-config.json

```diff
@@ -1,5 +1,6 @@
 {
-	"navconf": "$HOME/Library/Application\ Support/NavCoin4/navcoin.conf",
-	"runningNavVersion": "4.1.1",
-	"detectedIp": ""
-}
+	"navconf": "",
+	"runningNavVersion": "",
+	"allowedIps": null,
+	"uiPassword": "$2a$14$TmLKDezfTgOJzMy3YXQ3C.AYj8mD7EEVpimO6NceBiVlr2TM9ds1y"
```

Comment:
@red010b37 should "uiPassword" be empty, or is this just to verify test?

Reply:
This is just to verify and show you what it looks like :)

Reply:
most excellent